src/styles/mixins.scss

```diff
@@ -88,3 +88,18 @@
   border-radius: 50%;
   outline: none;
 }
+
+@mixin paginationItem {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  box-sizing: border-box;
```

Comment:
border-box is set for all elements by default

Reply:
Done!